yarn-project/circuits.js/src/structs/index.ts

```diff
@@ -35,3 +35,10 @@ export * from '@aztec/foundation/eth-address';
 export * from '@aztec/foundation/fields';
 export * from '@aztec/foundation/aztec-address';
 export { FunctionSelector } from '@aztec/foundation/abi';
+
+// TODO(Kev): This is only exported so that privateKernelInit in noir-private-kernel
+// does not need to manually initialize its instance.
+// This is not great as we are exporting from tests.
+// Its okay for now and before merging into master, we should
+// remove this line.
+export {makePrivateKernelInputsInit} from '../tests/factories.js';
```

Comment:
I think this is okay tech debt at the moment -- until we want to merge into master